pkg/generate/config/config.go

```diff
@@ -321,6 +338,19 @@ func (c *Config) OverrideValues(operationName string) (map[string]string, bool)
 	return oConfig.OverrideValues, ok
 }
 
+// AdditionSpec gives map of operation and their MemberFields to
```

Comment:
s/AdditionSpec/SpecFieldConfig/

(plus, the description of what this method does is incorrect :) )